utils/textnorm_zh.py

```diff
@@ -35,6 +35,10 @@
 # SIL = [u'杠', u'槓']
 
 FILLER_CHARS = ['呃', '啊']
+ALLOW_ER_WRD='(儿女|儿子|儿孙|女儿|儿媳|妻儿|' \
```

Comment:
btw, can this be done via blacklist(positive Erhua hits) instead of current whitelist(negative Erhua hits)? Using a blacklist seems to be a more reasonable approach, we can improve the blacklist over time.